packages/flutter_list_bloc/test/continuous_listview_bloc_builder_test.dart

```diff
@@ -0,0 +1,249 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_list_bloc/flutter_list_bloc.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:list_bloc/list_bloc.dart';
+
+import 'helpers/item.dart';
+
+void main() {
+  group('ContinuousListViewBlocBuilder', () {
+    late ListCubit<Item, OffsetLimitFilter> listCubit;
```

Comment:
Also I think we should close the cubit after using.

Reply:
I have made the necessary changes to close the cubit in the widget tests as per your recommendation. I basically used the [tearDown](https://github.com/aniketambore/list_bloc/blob/widget-tests/packages/flutter_list_bloc/test/continuous_listview_bloc_builder_test.dart#L40) function in the widget tests which ensures that the cubit is properly closed and resources are cleaned up after each tests, maintaining a clean testing environment. Thank you for your suggestion!